src/include/reconcile.h

```diff
@@ -66,6 +72,17 @@ struct __wt_rec_chunk {
     uint32_t auxentries;
 };
 
```

Comment:
These need a comment, I will do that after the latest round of patch testing has finished.